test/task/NestedMultisigTask.t.sol

```diff
@@ -60,17 +60,18 @@ contract NestedMultisigTaskTest is Test {
         IMulticall3.Call3Value[] memory calls = new IMulticall3.Call3Value[](1);
         calls[0] = call;
 
-        /// callDataToApprove is the data that the child multisig has to execute to
-        /// approve the transaction that the parent multisig is going to execute.
+        // callDataToApprove is the data that the child multisig has to execute to
+        // approve the transaction that the parent multisig is going to execute.
         bytes memory callDataToApprove =
             abi.encodeWithSignature("aggregate3Value((address,bool,uint256,bytes)[])", calls);
         assertEq(callDataToApprove, multisigTask.generateApproveMulticallData(), "Wrong callDataToApprove");
         for (uint256 i; i < childOwnerMultisigs.length; i++) {
-            /// dataToSign is the data that the EOA owners of the child multisig has to sign to help
-            /// execute the child multisig approval of hashToApproveByChildMultisig
+            // dataToSign is the data that the EOA owners of the child multisig has to sign to help
+            // execute the child multisig approval of hashToApproveByChildMultisig
             bytes memory dataToSign = getNestedDataToSign(childOwnerMultisigs[i]);
-            /// nonce is not decremented by 1 because in task simulation approveHash is called by
-            /// the child multisig which does not increment the nonce
+            // nonce is not decremented by 1 because in task simulation approveHash is called by
+            // the child multisig which does not increment the nonce
+            uint256 nonce = IGnosisSafe(childOwnerMultisigs[i]).nonce();
```

Comment:
@ElliotFriedman @blmalone @prateek105 I pulled out nonce into a variable here to mitigate a stack too deep error. Please confirm this is safe to reuse in this way, i.e. I assume the nonce is expected to be the same for both the `encodeTransactionData` and `getTransactionHash` calls?